Climate_police/tests/hw7-mao/pollution_map.py

```diff
@@ -0,0 +1,44 @@
+import plotly.offline as py
+py.init_notebook_mode()
+
+# import pandas as pd
+
+from pre_process import pre_process
+
+def pollution_map(df, source, year):
+	
+	# Pre-processes the pollution data so that it can be plotted by plotly.
+	df2 = pre_process(df)
+
+	scl = [[0.0, 'rgb(242,240,247)'],[0.2, 'rgb(218,218,235)'],[0.4, 'rgb(188,189,220)'],\
+            [0.6, 'rgb(158,154,200)'],[0.8, 'rgb(117,107,177)'],[1.0, 'rgb(84,39,143)']]
+
+	data = [ dict(
+        	type='choropleth',
+        	colorscale = scl,
+        	autocolorscale = False,
+        	locations = df2['abbrev'],
+        	z = df2[source+' Mean'].astype(float),
```

Comment:
there can be an option for users to select if they want to see mean value or maximum value